<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZTDemand</class>
 <widget class="QWidget" name="ZTDemand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="productFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelProduct">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxProduct">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="moduleFrame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelModule">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxModule">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="planFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>64</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelPlan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxPlan">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="planFrame_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>64</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelSource">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxSource">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSourceNote">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSourceNote">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>90</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="reviewFrame">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>108</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>32</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="boxReview">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>290</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>290</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelReview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="boxNoReviewer">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxReviewer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelReviewer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="ursFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>152</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelURS">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxURS">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="parentFrame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>152</y>
     <width>350</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelParent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxParent">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="demandFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>196</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelDemand">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDemand">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>290</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelType">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxType">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>55</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelPri">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="boxPri">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>0</y>
      <width>55</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelStory">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="descFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>720</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelDesc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>660</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="verifyFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>432</y>
     <width>720</width>
     <height>180</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelVerify">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="texVerify">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>660</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fileFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>624</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelFileLimit">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="maintoFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>668</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelMainto">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxMailto">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>580</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxContact">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>0</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="keywordFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>712</y>
     <width>720</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelKeyword">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editKeyword">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>660</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
